thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/controller.js

```diff
@@ -88,14 +112,18 @@ export default Controller.extend({
     function() {
       const severityMap = this.get('severityMap');
       const patternMap = this.get('patternMap');
+      const amountChange = this.get('customPercentChange');
       const selectedPattern = this.get('selectedTunePattern');
       const selectedSeverity = this.get('selectedSeverityOption');
```

Comment:
You should be able to chain these in one line like such:
`const { severityMap, patternMap, amountChange, ... } = this;`